caffe2/python/operator_test/mean_op_test.py

```diff
@@ -29,14 +29,16 @@
 
 class TestMean(hu.HypothesisTestCase):
     @given(
-        k=st.integers(1, 5),
-        n=st.integers(1, 10),
-        m=st.integers(1, 10),
+        k=st.integers(2, 2),
```

Comment:
Wait, why did these change? k, n amd m should stay the previous way.

Reply:
These changes were accidental; specific test cases were tested for validating k gradient outputs. Reverting these.